typescript-conditionals/main.ts

```diff
@@ -0,0 +1,116 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number: number): boolean {
+  return number < 5;
+}
+console.log('isUnderFive: ', isUnderFive(3));
+/// /////////////////////////////////////////
+function isEven(number: number): boolean {
+  return number >= 2 + 2;
+}
```

Comment:
This isn't accounting for even numbers, this is just checking if `number` is greater than or equal to 4. You should be utilizing the `%` operator, to check if a number is even or odd